### PR TITLE
[stable/prometheus-redis-exporter] Adding abbility to use custom script

### PR DESCRIPTION
**Pull request by alexzimmer96**
#### What this PR does / why we need it:
* The used container included the ability to run custom lua scripts by setting an environment variable (`REDIS_EXPORTER_SCRIPT`)
* This PR allows to specify a script from a ConfigMap and mount it into the container
* `REDIS_EXPORTER_SCRIPT` will be set when a script is used

#### Checklist
- [X] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/prometheus-redis-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -22,12 +22,6 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ template "prometheus-redis-exporter.serviceAccountName" . }}
-      {{- if .Values.image.pullSecrets }}
```
Comment (CVirus) [inline stable/prometheus-redis-exporter/templates/deployment.yaml]: Why did you remove this?

Comment (alexzimmer96) [reply stable/prometheus-redis-exporter/templates/deployment.yaml]: Hi CVirus,
i've corrected this. We made some changes yesterday without having the project forked. Today, we just copied the files into our forked version and this has been overriden.

I've added the imagePullSecret-Parts back to the deployment. Thanks for your feedback.

**Comment by acondrat** [comment]
looks good to me 
/ok-to-test
